### PR TITLE
Release 0.2.0 examples

### PR DESCRIPTION
**Pull request by awholegunch**
Various and sundry doc-related changes.
Staged at http://stg-web101.sw.joyent.continuuity.net/tigon/0.2.0-SNAPSHOT/en/index.html

### CODE REVIEW

File: tigon-docs/developer-guide/source/faq.rst [hunk 1]
```diff
@@ -54,11 +54,11 @@ Running Tigon
 
 Download the tar.gz, untar it and checkout the examples!
```
Comment (gokulavasan) [inline tigon-docs/developer-guide/source/faq.rst]: Download the zip, unzip it....

**Comment by gokulavasan** [comment]
LGTM :+1:

**Comment by gokulavasan** [comment]
@awholegunch We can remove the libcurl requirement from all places. That PR has been merged. After that we can merge this PR.

**Comment by awholegunch** [comment]
Please re-review this. An update to the staging server shows the completed docs:
http://stg-web101.sw.joyent.continuuity.net/tigon/0.2.0-SNAPSHOT/en/index.html

**Comment by gokulavasan** [comment]
LGTM :+1:

**Pull request merged**